public/javascripts/registration-validation.js

```diff
@@ -0,0 +1,61 @@
+var errorNames = [];
+var validator = new FormValidator('registration_form', [{
+  name: 'firstname',
+  display: 'required',
+  rules: 'required|alpha'
+},
+  {
+    name: 'lastname',
+    display: 'required',
+    rules: 'required|alpha'
+  },
+  {
+    name: 'password',
+    rules: 'required|callback_check_password'
+  },
+  {
+    name: 'repeatpassword',
+    display: 'repeated password',
+    rules: 'required|matches[password]'
+  },
+  {
+    name: 'email',
+    rules: 'required|valid_email'
+  },
+  {
+    name: 'terms',
+    display: 'terms &amp; conditions',
+    rules: 'required'
+  }
+], function(errors, event) {
+  for(var j=0; j<errorNames.length; j++){
+    document.getElementById(errorNames[j]+'_error').setAttribute('style','display:none;');
+  }
+  errorNames = [];
```

Comment:
Here we hide all errors to show them again? 
```
for(var j=0; j<errorNames.length; j++){
 document.getElementById(errorNames[j]+'_error').setAttribute('style','display:none;');
}
```
why can't we use at least classes for that?